supply/lib/supply/uploader.rb

```diff
@@ -162,8 +162,12 @@ def update_track(apk_version_codes)
       end
     end
 
+    # returns only language directories from metadata_path
     def all_languages
-      Dir.foreach(metadata_path).sort { |x, y| x <=> y }
+      Dir.entries(metadata_path)
```

Comment:
@KrauseFx This _should not_ have any negative side effect. The existing setup was already breaking since this was picking up an HTML file 😉